code/espurna/sensors/BME680Sensor.h

```diff
@@ -0,0 +1,242 @@
+// -----------------------------------------------------------------------------
+// BME680 Sensor over I2C
+// Copyright (C) 2017-2019 by Xose Pérez <xose dot perez at gmail dot com>
```

Comment:
👋
2020 @ruimarinho

I'd also link to the BSEC repo

Reply:
I think we should make this more legally robust. Something like:

```
// -----------------------------------------------------------------------------
// BME680 Sensor over I2C
// Copyright (C) 2017-2020 by Xose Pérez <xose dot perez at gmail dot com>
//
// The BSEC software binaries and includes are only available for use after accepting its software
// license agreement. By enabling this sensor integration, you are agreeing to the terms of the license
// agreement available at the following URL:
//
// https://ae-bst.resource.bosch.com/media/_tech/media/bsec/2017-07-17_ClickThrough_License_Terms_Environmentalib_SW_CLEAN.pdf
//
// The Arduino wrapper and BME680 Sensor API used for this integration are licensed under the following terms:
//
//   Copyright (c) 2020 Bosch Sensortec GmbH. All rights reserved.
//
//   BSD-3-Clause
//
//   Redistribution and use in source and binary forms, with or without
//   modification, are permitted provided that the following conditions are met:
//
//   1. Redistributions of source code must retain the above copyright
//       notice, this list of conditions and the following disclaimer.
//
//   2. Redistributions in binary form must reproduce the above copyright
//      notice, this list of conditions and the following disclaimer in the
//      documentation and/or other materials provided with the distribution.
//
//   3. Neither the name of the copyright holder nor the names of its
//      contributors may be used to endorse or promote products derived from
//      this software without specific prior written permission.
//
//   THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
//   "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
//   LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
//   FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
//   COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
//   INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
//   (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
//   SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
//   HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
//   STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
//   IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
//   POSSIBILITY OF SUCH DAMAGE.
//
// For more details, please refer to https://github.com/BoschSensortec/BSEC-Arduino-library.
// -----------------------------------------------------------------------------
```
WDYT?

Reply:
ref. https://www.gnu.org/licenses/gpl-howto.en.html
As you have modified / created the file, the intent is to have a note about that.

.pdf agreement notice is important. (*ianal*) BSD only applies to Arduino integration code though?

Reply:
Sorry, I don't think I follow here. My understanding is that the Arduino integration code is BSD while the BSEC binary uses a proprietary license that needs to be accepted by the end-user in order to use it.

Are you suggesting something different from what I have added? I believe I am attaching the copyright notice of the Arduino integration (BSD), as requested by the license, and linking to the PDF for the BSEC agreement.

Is your reference specific to `libalgobsec.a`?

Reply:
My main question is regarding redistribution part, as we don't really intend to publish the .bin with bme680 support

Reply:
As long as copyright is included, isn't redistribution of the Arduino library ok?